requirements.txt

```diff
@@ -25,3 +25,4 @@ conllu>=4.0
 more-itertools
 wikipedia-api
 pptree
+sacremoses
```

Comment:
This package was not installed when I changed from Python 3.6 to 3.7

Reply:
Yes it is no longer a dependency of the `transformers` library which also deprecated support for 3.6. We removed the class that requires `sacremoses` so adding the dependency should not be required.

Reply:
actually, without `sacremoses` I noticed an import error from `transformers` library.

Reply:
this should be fixed in master branch, see #2768 - is your error on the latest Flair release or on current master branch state?

Reply:
I got [this error](https://github.com/aniketmaurya/flair/runs/6464440062?check_suite_focus=true) in the Github CI, so it would be the current master branch state.

Reply:
I think your branch is behind current master by a few commits. It was fixed by one of the last commits.

Reply:
oh yes, sorry! Just merged with master.